package study.errors;

public class throwDemo {

	public static void main(String[] args) {
		
		System.out.println(10/0);  //Arithmetic Exception by JVM
		
		//throw new ArithmeticException("Divided By Zero");
	}
}
